### PR TITLE
Add WasmEdge-en

### PR DESCRIPTION
**Pull request by alabulei1**
Signed-off-by: alabulei1 vivian.xiage@gmail.com

### CODE REVIEW

File: versioned_docs/version-v0.6.0/user-manuals/runtime/WasmEdge.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
+# Use OpenYurt to Manage WasmEdge
+
+In this article, we will cover the steps to install and set up the entire toolchain for running a [WasmEdge application in OpenYurt Cluster.
+
+[WasmEdge](https://wasmedge.org/)(Project under CCNF) is a [OCI-compliant and k8s compatible](https://wasmedge.org/book/en/kubernetes.html) lightweight WebAssembly runtime. It provides an alternative to [Linux containers](https://wasmedge.org/wasm_linux_container/) for containerized apps on edge. WasmEdge provides capability-based secure sandboxes, makes a very little assumption about the underlying OS services, and at the same time, supports [multiple programming languages](https://wasmedge.org/book/en/dev.html) on the front end.
```
Comment (JcJinChen) [inline versioned_docs/version-v0.6.0/user-manuals/runtime/WasmEdge.md:5]: replace CCNF to CNCF?

File: versioned_docs/version-v0.6.0/user-manuals/runtime/WasmEdge.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
+# Use OpenYurt to Manage WasmEdge
+
+In this article, we will cover the steps to install and set up the entire toolchain for running a [WasmEdge application in OpenYurt Cluster.
```
Comment (JcJinChen) [inline versioned_docs/version-v0.6.0/user-manuals/runtime/WasmEdge.md:3]: remove [ in [WasmEdge?

**Comment by rambohe-ch** [comment]
@alabulei1 Very appreciate for adding docs of WasmEdge runtime. please following the comments of chinese version docs and fix them.

File: versioned_docs/version-v0.6.0/user-manuals/runtime/WasmEdge.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+
+```
+sudo swapoff -a
+//verify    
+free -m
+```
+
+Next, let’s configure the file `/etc/hosts` of two nodes.
+
+![](configure-node.png)
```
Comment (JcJinChen) [inline versioned_docs/version-v0.6.0/user-manuals/runtime/WasmEdge.md:19]: could you move this pictures as required?

**Review by JcJinChen — CHANGES_REQUESTED** [review]

File: versioned_docs/version-v0.6.0/user-manuals/runtime/WasmEdge.md (hunk 3 above)
Comment (JcJinChen) [reply versioned_docs/version-v0.6.0/user-manuals/runtime/WasmEdge.md:19]: https://github.com/openyurtio/openyurt.io#for-historic-version-example-v050

**Comment by alabulei1** [comment]
Thanks for your feedback. Please check out again.

File: versioned_docs/version-v0.6.0/user-manuals/runtime/WasmEdge.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+wget https://github.com/containerd/containerd/releases/download/v${VERSION}/cri-containerd-cni-${VERSION}-linux-amd64.tar.gz
+wget https://github.com/containerd/containerd/releases/download/v${VERSION}/cri-containerd-cni-${VERSION}-linux-amd64.tar.gz.sha256sum 
+sha256sum --check cri-containerd-cni-${VERSION}-linux-amd64.tar.gz.sha256sum
+
+sudo tar --no-overwrite-dir -C / -xzf cri-containerd-cni-${VERSION}-linux-amd64.tar.gz
+sudo systemctl daemon-reload
+```
+
+
+As the crun project support WasmEdge as default, we just need to configure the containerd configuration for runc. So we need to modify the runc parameters in `/etc/containerd/config.toml` to curn and add `pod_annotation`.
```
Comment (JcJinChen) [inline versioned_docs/version-v0.6.0/user-manuals/runtime/WasmEdge.md:70]: replace curn to crun?

**Pull request merged**